### DISCUSSION TITLE
How to use internalComHost and internalComPort when use nacos as skywalking Cluster Management

### DISCUSSION THREAD

**Discussion by fruitfish** [apache/skywalking#10677] [Q&A]
I use nacos as skywalking Cluster Management. 
question 1:
Are internalComHost and internalComPort must be set.
question2:
Can internalComPort be set any value, such as 30000, or must be equal  to gRPCPort.

**Comment by wu-sheng** [comment]
If you don't know its meaning, you don't need them. They are optional only for rare cases
